tests/boot/boot_from_pxe.pm

```diff
@@ -99,6 +99,7 @@ sub run {
     }
     if (check_var('BACKEND', 'ipmi')) {
         $image_path .= "ifcfg=$interface=dhcp4 " unless get_var('NETWORK_INIT_PARAM');
+        $image_path .= "ipv6.disable=1 " if get_var('LINUX_BOOT_IPV6_DISABLE');
```

Comment:
Awesome @waynechen55! Would you mind adding it to: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md? thanks!

Reply:
Done